aws/modules/deploy/helms/gpii-preferences/README.md

```diff
@@ -0,0 +1,42 @@
+# GPII Preferences
+
+Here goes the service description.
```

Comment:
Update this. Link to gpii.net? github.com/gpii? gpii/universal?

Reply:
Updated.